tests/build_image_base_test.py

```diff
@@ -94,7 +94,13 @@ def test_sam_init(self):
         """
         Test sam init hello world application for the given runtime and dependency manager
         """
-        if self.runtime in ["provided", "provided.al2", "provided.al2023", "dotnet7", "dotnet9"]:
+        if self.runtime in [
+            "provided",
+            "provided.al2",
+            "provided.al2023",
+            "dotnet7",
+            "dotnet9",
+        ]:
```

Comment:
We should be able to also skip for new runtimes that are not available in `sam init` yet so the test don't fail.